frame/staking/src/pallet/impls.rs

```diff
@@ -674,6 +692,13 @@ impl<T: Config> Pallet<T> {
 		<ErasStakersClipped<T>>::remove_prefix(era_index, None);
 		#[allow(deprecated)]
 		<ErasValidatorPrefs<T>>::remove_prefix(era_index, None);
+		#[allow(deprecated)]
```

Comment:
Someday we should tackle this and remove all of this in a semi-lazy way, with the help of the correct removal api.